src/shared/seccomp-util.h

```diff
@@ -115,3 +118,27 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(scmp_filter_ctx, seccomp_release);
 int parse_syscall_archs(char **l, Set **ret_archs);
 
 uint32_t scmp_act_kill_process(void);
+
+/* This is a special value to be used where syscall filters otherwise expect errno numbers, will be
+   replaced with real seccomp action. */
+typedef enum SeccompErrorNumber {
+        SECCOMP_ERROR_NUMBER_KILL = INT_MAX - 1,
+} SeccompErrorNumber;
+
+static inline bool seccomp_errno_or_action_is_valid(int n) {
+        return n == SECCOMP_ERROR_NUMBER_KILL || errno_is_valid(n);
+}
+
+static inline int seccomp_parse_errno_or_action(const char *p) {
+        if (streq(p, "kill"))
+                return SECCOMP_ERROR_NUMBER_KILL;
+        return parse_errno(p);
+}
+
+static inline const char *seccomp_errno_or_action_to_string(int num) {
+        if (num == SECCOMP_ERROR_NUMBER_KILL)
+                return "kill";
+        if (num >= 0)
```

Comment:
you can drop this if check and unconditionally cal errno_to_name() here, no?

Reply:
That would accept both negative and positive errno codes. Maybe that's expected though.

Reply:
changed